src/main.rs

```diff
@@ -81,9 +81,9 @@ async fn main() -> Result<()> {
     let app = App::parse();
 
     match app.command {
-        Command::New { path } => new_account(path)?,
+        Command::New { path } => new_account_cli(path)?,
```

Comment:
Seeing as these `path` arguments always eventually converted from `Option<String>` to some form of `Path`, is it possible to change our `Command` variants to have `Option<PathBuf>` rather than `Option<String>`? Can clap still accept strings on the command line in this case?